posydon/binary_evol/binarystar.py

```diff
@@ -162,6 +162,11 @@ def __init__(self, star_1=None, star_2=None, index=None, properties=None,
             self.inspiral_time = None
         if not hasattr(self, 'mass_transfer_case'):
             self.mass_transfer_case = 'None'
+
+        if not hasattr(self, 'true_anomaly_first_SN'):
```

Comment:
For consistency, this quantity should be stored only once for each star in the star object as a fith value to the the natal kick array. Given that this will be the fifth value associated to the kicks, to avoid confusions, maybe we should get rid of the natal kick array and have proper names for each variable stored separatly. Also in the future the user might not want to store the entire natal kick array, but say, e.g. only the magnitude of the kick.

Reply:
I never liked the natal kick array, but until we decide to make a change to it, we are stuck with it. Nevertheless, I think it is fine to keep it as is (with just the four parameters) and add these two new variables to the binary object. We may want to come up with a better option in the future, but for now, I'm fine with this nomenclature.